### DISCUSSION TITLE
Can someone explain the proper usage of REPLACE_ME_WITH_APP_CENTER_SECRET

### DISCUSSION THREAD

**Discussion by tephyrnex** [1Remote/1Remote#728] [_Development]
I always like to compile my own version of OS projects like this when possible; however, I'm not clear what type of secret should be put into the build script to successfully compile.  Any help is appreciated.  Thanks!

**Comment by VShawn** [comment]
"REPLACE_ME_WITH_APP_CENTER_SECRET" is used for integrating with Microsoft's bug collection system. You simply need to clear the `PreBuildEvent` and `PostBuildEvent`, then you can ignore this parameter.

https://github.com/1Remote/1Remote/blob/main/Ui/Ui.csproj#L497-L503
